boot-cloverage/src/cloverage/boot_cloverage.clj

```diff
@@ -0,0 +1,36 @@
+(ns cloverage.boot-cloverage
+  (:require [boot.core :as core]
+            [boot.pod :as pod]
+            [boot.util :as util]
+            [clojure.string :as s]
+            [clojure.set :as set]))
+
+(def ^:private deps
+  '[[cloverage "1.0.8-SNAPSHOT"]])
+
+(core/deftask cloverage
+  "Run cloverage in a pod.
+
+  Expects test namespaces to be included in :source-paths."
+
+  [m test-matcher VAL regex "Regex used to select test namespaces"
+   o opts         VAL str   "other arguments passed to cloverage"]
+  (let [pod (-> (core/get-env)
+                (update-in [:dependencies] into deps)
+                pod/make-pod
+                future)]
+    (core/with-post-wrap fileset
+      (let [namespaces (core/fileset-namespaces fileset)
+            test-matcher-or-default (or test-matcher #".*-test")
+            [code-namespaces test-namespaces] ((juxt remove filter) #(re-matches test-matcher-or-default (name %)) namespaces)
```

Comment:
i guess it's a matter of style since you then have to destructure with true/false. i don't feel strongly either way. this seemed to be idiomatic after looking around.